src/applications/coronavirus-research/update/config/submitForm.js

```diff
@@ -34,30 +34,38 @@ const submitForm = (form, formConfig, submissionId) => {
     return value ? 'Yes' : 'No';
   };
 
+  // format values for Analytics
   const eventData = {
-    'previously-diagnosed-with-covid19': formData.diagnosed ? 'Yes' : 'No',
-    'close-contact-with-positive-test': getFormattedRadioValues(
-      'closeContactPositive',
-      formData.closeContactPositive,
-    ),
-    'hospitalized-w-in-six-months': resolveBooleanValue(formData.hospitalized),
-    'smoke-or-vape-history': resolveBooleanValue(formData.smokeOrVape),
-    'historical-health-issues': getFormattedTrueSelectValues('HEALTH_HISTORY'),
-    'work-situation': getFormattedTrueSelectValues('EMPLOYMENT_STATUS'),
-    'work-transportation': getFormattedTrueSelectValues('TRANSPORTATION'),
-    'people-living-at-home': getFormattedRadioValues(
-      'residentsInHome',
-      formData.residentsInHome,
-    ),
-    'close-contact-outside-of-home': getFormattedRadioValues(
-      'closeContact',
-      formData.closeContact,
-    ),
-    'zip-code': formData.zipCode,
-    'relationship-to-va': getFormattedTrueSelectValues('VETERAN'),
-    gender: getFormattedTrueSelectValues('GENDER'),
-    'race-ethnicity-origin': getFormattedTrueSelectValues('RACE_ETHNICITY'),
     previousSubmissionId: submissionId,
```

Comment:
i know this isn't affected by this PR, but just want to check — is `previousSubmissionId` being carried all the way through to GenISIS? if so, is it possible to change to `registryUUID` which would standardized it with what's already in GenISIS.

Reply:
yes the naming here carries over to Genisis so we will change it here and in schemas. Thanks!